tools/SchemaManager/SchemaDataStore.cs

```diff
@@ -30,11 +30,7 @@ public static void ExecuteScriptAndCompleteSchemaVersion(string connectionString
 
                     serverConnection.BeginTransaction();
 
-                    // Since version 3 is just the base schema which must already be executed at the start of apply command and can be skipped
```

Comment:
This is not related to the fix but identified this version 3 specific to fhir-server should be removed. As this is the common tool across dicom, fhir and any future repos.